Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM php:8.2.7-cli-alpine3.18 AS build
+FROM php:8.3.3-cli-alpine3.18 AS build
```

Comment:
I would even opt for using `php:8.3-cli-alpine3.18` instead so that we would not need to maintain on ever minor PHP update.